docs/docs/code/nodejs/README.md

```diff
@@ -1,26 +1,18 @@
-# Code
```

Comment:
Many of the code in this doc won't work in Node.js steps in bob, since they run in the component code environment. A few of the specific changes you should make:

* [Adding a code step section](https://pipedream-docs-redirect-do-not-edit-git-doc-b8b90e-pipedreamers.vercel.app/docs-v2/code/nodejs/#adding-a-code-step) — screenshot / instructions will need to be modified, the code example with `this.test` doesn't work, etc.
* [Passing parameters to code steps](https://pipedream-docs-redirect-do-not-edit-git-doc-b8b90e-pipedreamers.vercel.app/docs-v2/code/nodejs/#passing-parameters-to-code-steps) section needs to reference props instead of params (we should probably have a simple example of a Node.js component with a prop here)
* [async function declaration](https://pipedream-docs-redirect-do-not-edit-git-doc-b8b90e-pipedreamers.vercel.app/docs-v2/code/nodejs/#async-function-declaration) section should be removed. we may want to replace it with a section that discusses how `steps` is passed to the `run` method of components to show users how to access event data in here.
* We should also add a new section on how to link apps to steps via app props in the component, how that surfaces the connected account section of the step, etc. That section should probably live [in this doc](https://pipedream.com/docs/workflows/steps/code/auth/#the-auths-object), and we can link to that doc from here.
* We should remove the screenshot in the **Logs** section and change the language to note that logs are under the Results > Logs section of the output.

Reply:
The link to the updated `$.respond()` docs still references `$respond` (no `.`)

Reply:
The link to the docs on `$.send.http` still reference `$send` (also no `.`)

Reply:
The "managing state" section links to a 404 (since you removed the original `$checkpoint` doc). I would actually recommend keeping that doc and putting it under the Node.js sub-section, and just re-working the examples to use `$.service.db`. More comments on that below.

Reply:
Rework the `$end` section to use `$.flow.exit`

Reply:
[Using secrets in code](https://pipedream-docs-redirect-do-not-edit-git-doc-b8b90e-pipedreamers.vercel.app/docs-v2/code/nodejs/#using-secrets-in-code) section refers to making workflows public, so we'll want to clean up that section.

Reply:
I see what happened with some of the links. It looks like you `git mv`d the files but didn't update some of the references here (e.g. to the workflow state doc).